src/components/AppToolbar.vue

```diff
@@ -0,0 +1,29 @@
+<script lang="ts" setup>
+import { ref } from "vue";
+import { useRoute } from "vue-router";
+
+const route = useRoute();
+
+// TODO: replace placeholder profile menu
```

Comment:
I wouldn't know what to do when reading this todo. Replace it by what ..?

Reply:
Yeah, kinda vague haha. I just added a basic dropdown placeholder for the profile menu/dropdown. This will be implemented in a separate PR

Reply:
Is it better to link to the github issue? #8